src/api/subscriptions.js

```diff
@@ -236,6 +236,17 @@ export class Subscriptions {
    */
   attachButton(button, optionsOrCallback, opt_callback) {}
 
+  /**
+   * Creates smartButton element and the provided callback.
+   * The default theme is "light".
+   *
+   * @param {!Element} container
+   * @param {!ButtonOptions=} opt_options
+   * @param {function()=} opt_callback
+   * @return {!Promise}
+   */
+  renderSmartButton(container, opt_options, opt_callback) {}
```

Comment:
Let's also discuss the naming of `attachSmartButton`. The example above is pretty similar from many points of view. We might have a more obvious API if we move in that path.

Reply:
Done as discussed offline